docs/tutorials/intro.md

```diff
@@ -0,0 +1,177 @@
+- In a similar way, you can edit the symbology of the first Shapefile Layer we added and change the **Fill Color**, **Stroke Color** and **Stroke Width**.
+- Do you still see the roads in New-York? Try to adjust the **Opacity** value (default is 1) to a lower value for this Shapefile Layer. Can you see all your layers now?
+
+```{exercise-end}
+
+```
+
+```{solution} add-vector-layer
+:class: dropdown
+
+After adding the new Shapefile Layer and zooming over New-York, you should have the following map:
+![View several shapefile Layer on the same map](../assets/images/severalShapefileLayers.png)
+
+When you right click to edit the symbology, you should get the following pop-up menu:
+![Edit symbology](../assets/images/editSymbology.png)
+
+And when editing the symbology of the other Shapefile Layer:
+![Adjust Symbology](../assets/images/adjustSymbology.png)
+
+And updating the opacity (for instance to 0.5) of the US county Shapefile Layer, you can get a map that can look like to (depending on the colors and Stroke width you chose!):
+
+![final map for the intro exercise](../assets/images/finalIntroMap.png)
+```
+
+## Hiding and Reordering Layers
+
+### Hiding/Viewing Layers
+
+In the GIS Layers List/Browser Panel, you can select a layer and click the **eye** icon to hide or show the corresponding layer.
+![Hide a layer](../assets/images/hideLayer.png)
+
+### Reordering Layers
+
+The layers in your Layers List are displayed on the map according to their order in the list. The bottom layer is drawn first, while the top layer is drawn last. You can adjust their drawing order by rearranging the layers in the list.
+
+The order in which the layers have been loaded into the map is probably not logical at this stage. It’s possible that the road layer or/and the country layer are completely hidden because other layers are on top of it.
+
+For example, this layer order:
+![Wrong layer order](../assets/images/wrongLayerOrder.png)
+
+would hide all the custom shapefile layers.
+
+To solve this problem, you can click and drag on a layer in the Layers list to move it up or down, and reordering the layers as you wish them to be viewed on the map.
+
+## Renaming Layers
+
+As shown on the previous image, the two custom shapefile layers we added have the same name in the GIS Layers List / Browser Panel which can be very confusing.
+It is a good practice to rename your layers with meaningful names. To rename a layer, select it in the GIS Layers List / Browser Panel, and right click to **Rename Layer**.
+
+```{exercise-start}
+:label: Rename-layer
+```
+
+- Rename each **Custom Shapefile Layer** with a meaningful name e.g., **cb_2015_us_county_500k** for the US County Shapefile Layer and **nyc_roads** for the roads in New-York.
```

Comment:
Layer names can have spaces in them, so I feel it would be valuable to write these as more human-readable names, like "NYC roads" and "US counties" instead of naming them after the input files. It might be useful to demonstrate renaming the _sources_ to match the input file names!